CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/)
 
+## 20240725
+- al2 ami version: 20240725
```

Comment:
Don't we need to add information about the source amis, like in the previous change log entries?